src/classes/storage.ts

```diff
@@ -0,0 +1,36 @@
+import { StopGroup, StopCluster, StopGroupGpuid, StopClusterGpuid } from '../types';
+
+/**
+ * @description Manipulate GroundPlaces.
+ */
+export class Storage {
+  /**
+   * @description Returns the stopGroup identified by its Gpuid.
+   * @param {StopGroupGpuid} stopGroupGpuid - Ground Place unique identifier of the StopGroup to find.
+   * @returns {StopGroup|undefined}
+   */
+  // @ts-ignore
+  public getStopGroupByGpuid(stopGroupGpuid: StopGroupGpuid): StopGroup | undefined {}
+
+  /**
+   * @description Returns the stopCluster identified by its Gpuid.
+   * @param {StopClusterGpuid} stopClusterGpuid - Ground place unique identifier of the StopCluster to find.
+   * @returns {StopCluster|undefined}
+   */
+  // @ts-ignore
+  public getStopClusterByGpuid(stopClusterGpuid: StopClusterGpuid): StopCluster | undefined {}
```

Comment:
undefined or void ?

Reply:
For me this can be `undefined` when the `stopClusterGpuid` is not found inside the GroundPlaces list

Reply:
should throw an error

Reply:
Updated